tests/symbolicator/test_sourcemap.py

```diff
@@ -106,4 +110,4 @@ def test_symbolicator_roundtrip(self):
 
         event = self.post_and_retrieve_event(event_data)
 
-        insta_snapshot_javascript_stacktrace_data(self, event.data)
+        insta_snapshot_javascript_stacktrace_data(insta_snapshot, event.data)
```

Comment:
the interesting part here is that because of parameterized tests, we get two different snapshot files. Which is a pain to manually review, and there is no hard assertion that both are matching